package c.exception;

public class MultiCatchSample {
    public static void main(String[] args) {
        MultiCatchSample sample = new MultiCatchSample();
        // sample.multiCatch();
    }

    public void multiCatchThree()
    {
        
    }
    /*
     * public void multiCatch() {
     * int intArray[] = new int[5];
     * try {
     * 
     * 
     * catch(Exception e)
     * {
     * System.out.println(intArray.length);
     * }
     * catch(ArrayIndexOutOfBoundsException e)
     * {
     * System.out.println("occured");
     * }
     * // catch (ArrayIndexOutOfBoundsException e) {
     * // System.out.println("ArrayIndexOutOfBoundsException occured");
     * // } catch (Exception e) {
     * // // TODO: handle exception
     * // System.out.println(intArray.length);
     * // }
     * }
     */

}
